widgets/ProgressDialog.cpp

```diff
@@ -19,14 +19,21 @@
 #include <QApplication>
 #include <QTimer>
 
-ProgressDialog::ProgressDialog(QString message, bool cancellable,
-                               int timeBeforeShow, QWidget *parent) : 
+ProgressDialog::ProgressDialog(
+    QString message,
+    bool cancellable,
+    int timeBeforeShow,
+    QWidget *parent,
+    Qt::WindowModality modality
+) : 
     m_showTimer(0),
```

Comment:
Ah, I nitpicked about this formatting in my review of the associated svcore changes just now. I feel bad about that, because I've no objection to this in general and I use it in Javascript myself. It just isn't used in SV, so I probably ought to continue nitpicking despite feeling bad.